python/instrumentation/openinference-instrumentation-crewai/src/openinference/instrumentation/crewai/_wrappers.py

```diff
@@ -205,15 +212,36 @@ def __call__(
                 ),
             )
             try:
-                response = wrapped(*args, **kwargs)
+                crew_output = wrapped(*args, **kwargs)
+                usage_metrics = instance.usage_metrics
+                if isinstance(usage_metrics, dict):
+                    span.set_attribute(
+                        LLM_TOKEN_COUNT_PROMPT, int(usage_metrics.get("prompt_tokens", 0))
+                    )
+                    span.set_attribute(
+                        LLM_TOKEN_COUNT_COMPLETION, int(usage_metrics.get("completion_tokens", 0))
+                    )
+                    span.set_attribute(
+                        LLM_TOKEN_COUNT_TOTAL, int(usage_metrics.get("total_tokens", 0))
+                    )
```

Comment:
I don't think we should default to 0. If usage metrics does not have a given key, we should just not set the value, not assume 0. WDYT?